contracts/Dockerfile

```diff
@@ -4,17 +4,17 @@
 # FROM audius-contracts:latest as contracts
 # COPY --from=contracts /usr/src/app/build/contracts/ ./build/contracts/
 
-FROM node:10.16 as builder
+FROM node:16 as builder
```

Comment:
can we do node 14 for consistency with identity and creator node?

Reply:
I tried 14 already and sadly, it does not work

Reply:
Is there anything blocking us from upgrading to node 16 on identity and creator node longer term?

Reply:
longer term no, immediately there are some build errors. our applications don't work at the moment with node 16, at least they didnt when i tried 3-4 months ago